image_handler/image_generator.py

```diff
@@ -160,32 +182,43 @@ def stop_processing(self):
 
 
 # save image to database
-def save_image(image: Image.Image, info: ImageInfo):
-    url = "http://127.0.0.1:5000/image/create"
-    image_type = "jpeg"
-
-    image_bytes = io.BytesIO()
-    image.save(image_bytes, format=image_type)
-    image_bytes.seek(0)
-
-    files = {"file": (info.filename, image_bytes, f"image/{image_type}")}
-
-    req = requests.post(
-        url,
-        {
-            "real": info.real,
-            "date": info.date,
-            "theme": info.theme,
-            "status": info.status,
-        },
-        files=files,
-        timeout=10,
-    )
-    status = req.status_code
+def save_image(image: Union[Image.Image, str], info: ImageInfo):
+    if image is Image.Image:
+        image_type = "jpeg"
+        image_bytes = io.BytesIO()
+        image.save(image_bytes, format=image_type)
+        image_bytes.seek(0)
+
+        files = {"file": (info.filename, image_bytes, f"image/{image_type}")}
+
+        response = requests.post(
+            URL,
+            {
+                "real": False,
+                "date": info.date,
+                "theme": info.theme,
+                "status": ImageStatus.UNVERIFIED.value,
+            },
+            files=files,
+            timeout=10,
+        )
+    else:
+        response = requests.post(
+            URL,
+            {
+                "url": image,
+                "real": True,
+                "date": info.date,
+                "theme": info.theme,
+                "status": ImageStatus.UNVERIFIED.value,
+            },
+            files=None,
+        )
 
+    status = response.status_code
     if status == 200:
```

Comment:
```suggestion
    if response.ok:
```
nit: just a cleaner way to check for a successful response, will catch any codes that look like 2XX